python/pyspark/sql/connect/functions.py

```diff
@@ -551,6 +551,24 @@ def log(col: "ColumnOrName") -> Column:
     return _invoke_function_over_columns("ln", col)
```

Comment:
shall we remove this?

Reply:
(Python doesn't support overload so this function definition will simply be overwritten and no-op)